### PR TITLE
Fix typo of configuring redis at datadog initializer

### PR DESCRIPTION
**Pull request by JuanitoFatas**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Fix a typo for configuring [Redis](https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby/#redis).

## Related Tickets & Documents

- Related Issue #
- Closes #

## Added/updated tests?

- [ ] Yes
- [x] No need
- [ ] I need help with writing tests

### CODE REVIEW

**Review by rt4914 — APPROVED** [review]
Thanks @JuanitoFatas

**Comment by rt4914** [comment]
@maestromac Waiting for one more review.

**Review by maestromac — APPROVED** [review]
Nice catch. LGTM!

**Pull request merged**
